pages/language-tour.md

```diff
@@ -80,41 +74,52 @@ To run this program:
 - Run the command `valc Hello.val Greet.val -o hello`
 - Run the command `./hello`
 
-*Alternatively, you may put both source files in a folder, say `Sources`, and compile the program with `valc Sources -o hello`.*
+*Alternatively, you may put both source files in a subdirectory, say `Sources/`, and compile the program with `valc Sources -o hello`.*
 
 Note that `greet` need not to be `public` to be visible from another file in the module.
 All entities declared at the top level of a file are visible everywhere in a module, but not beyond that module's boundary.
+{% comment %}
+Do we need to say, "unless marked private?"
```

Comment:
You mean `public`?
We don't have a private accessor at the moment.

Reply:
Oh.  KISS, then.